lib/foreman_tasks_core/runner/action.rb

```diff
@@ -24,7 +24,8 @@ def run(event = nil)
 
       def finalize
         # To mark the task as a whole as failed
-        error! 'Script execution failed' if failed_run?
+        require 'pry'; binding.pry
```

Comment:
Hello `pry`, nice to meet you :) Not that I don't like you, but please, leave this PR before I call :policeman: :)